test/solver_tests/percolation_test.py

```diff
@@ -74,7 +74,7 @@ def test_NJ_negative_similarity(self):
         psolver = PercolationSolver(joining_solver=joining_solver)
         psolver.solve(p_tree)
 
-        expected_newick_string = "((1,3),(2,0,(4,5)));"
+        expected_newick_string = "((1,3),(0,2,(4,5)));"
```

Comment:
This is a good example for why it's nice to check the __topology__ of a tree rather than the newick string - the ordering within is arbitrary, so we can get unstable tests.